src/components/breadcrumbs/README.md

```diff
@@ -9,15 +9,54 @@
 {% endraw %}
 ```
 
-Call **breadcrumbs** like a normal function, set values for arguments `'breadcrumbs'` to alter its content.
+Call **breadcrumbs** like a normal function, set values for arguments `'title', 'url'` to alter its content.
```

Comment:
Isn't it an `options` argument, which is an array of object with `title` and `url` keys? That's not easy to convey though :/